packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -359,7 +358,17 @@ export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
       });
   }
 
-  setColor(colorString: string | HslaColor) {
+  setColor(colorString: string | HslaColor | undefined) {
```

Comment:
Would it be better to set value `null` instead?

Reply:
Both would also be useful😊

Reply:
I think it would make most sense to have `this.setColor(null)`

As far I know `undefined` is mainly when attributes shouldn't be set based on a condition.